tests/basilisp/testrunner_test.py

```diff
@@ -263,6 +265,35 @@ def test_fixtures_with_errors(
     result.assert_outcomes(passed=passes, failed=failures, errors=errors)
 
 
+def test_basilisp_test_noargs(pytester: pytest.Pytester):
+    runtime.Namespace.remove(sym.symbol("a.test-path"))
+
+    code = """
+    (ns tests.test-path
+      (:require
+       [basilisp.test :refer [deftest is]]))
+    (deftest passing-test
+      (is true))
+    """
+    pytester.makefile(".lpy", **{"./tests/test_path": code})
+
+    # I couldn't find a way to directly manipulate the pytester's
+    # `sys.path` with the precise control needed by this test, so we're
+    # invoking `basilisp test` directly as a subprocess instead ...
+    basilisp = shutil.which("basilisp")
+    cmd = [basilisp, "test"]
+    result = subprocess.run(cmd, capture_output=True, text=True, cwd=pytester.path)
+
+    print(f"\n\n--cmd--start: {' '.join(cmd)}")
+    print(f"\n\n--stdout--\n\n {result.stdout.strip()}")
+    print(f"\n\n--stderr--:\n\n {result.stderr.strip()}")
+    print(f"\n\n--cmd--end--: {' '.join(cmd)}\n\n")
```

Comment:
Is this necessary or was this just for debugging?

Reply:
Not strictly necessary, as the following `assert` on `stdout` will indicate any issues. However, I found it helpful for debugging since it also displays `stderr`, which would otherwise be hidden. Let me know if you want it removed (or commented out).

Reply:
I would prefer if you could remove it. I don't want tests emitting stuff to stdout if it is avoidable.

Reply:
Done, thanks